static/sidebar/js/mobile/modal/index.js

```diff
@@ -151,7 +160,9 @@
             data.runTime.loginid || data.runTime.loginId,
             function(data){
               setTimeout(function(){
+                checkProducts(data);
```

Comment:
you mean, code this inside the setTimeout function? can it be coded in a separate function and just call it instead? because i use it a couple of times, on video click and autoNext